pkg/oci/oci.go

```diff
@@ -90,6 +91,11 @@ func getLocalOciStore() (*oci.Store, error) {
 	return oci.New(defaultOciStore)
 }
 
+func getTimeFromAnnotations(annotations map[string]string) string {
```

Comment:
nit: What do you think about showing the time since the image was created (e.g `14 minutes ago`) similar to `docker` ?

Reply:
Done :)
Now it looks like this:
```
$ sudo -E ./ig image list
INFO[0000] Experimental features enabled                
REPOSITORY                           TAG                 DIGEST       CREATED
albantest.azurecr.io/mytest          latest              30e8fd4c86ed 10 minutes ago
```
Or:
```
$ sudo -E ./ig image list | cat # or using --no-trunc
INFO[0000] Experimental features enabled                
REPOSITORY TAG DIGEST CREATED
albantest.azurecr.io/mytest latest sha256:30e8fd4c86edb742969125f6ae71da065d3207498f5ce528d922c683904f8770 2024-03-19T13:59:31+01:00
```